src/collections/_documentation/guides/getting-started/index.md

```diff
@@ -0,0 +1,125 @@
+
+![Create a Project]({% asset guides/getting-started/08.png @path %})
+
+### > **What's in a Project?**
+
+A Project is a logical entity that connects between the errors in your application (or a part of it) to the team members assigned to that project. The **Project Settings** configuration determines
+
+- Which errors should be ingested into your Sentry account and which should be dropped, through [Inbound Filters](https://docs.sentry.io/accounts/quotas/#inbound-data-filters)
+- Who to notify, about what error, and how, using [Alert Rules](https://docs.sentry.io/workflow/notifications/alerts/)
+- Which errors should be automatically assigned to which member or team through [Issue Owners](https://docs.sentry.io/workflow/issue-owners/)
+- Custom rules to fine-tune [Event Grouping](https://docs.sentry.io/data-management/event-grouping/) into issues.
+
+Also, when displaying the **Issue Stream** and **Discover** views, the top-level filter bar considers the projects you are a member of by default. This way, developers are looking at information that is relevant to their work, and see the errors that they might need to jump on.
+
+![My Issue Stream]({% asset guides/getting-started/09.png @path %})
+
+### > **How Many Projects Should I Create?**
+
+You could theoretically pour all your errors into a single project. The Sentry UI will not limit you. However, considering the above ([What's in a Project?](#-whats-in-a-project)), setting up multiple projects to reflect your application landscape with finer granularity, will go a long way with regards to visibility, actionability, and productivity.
+
+Here are some points to consider:
+
+- If your app source code is managed in multiple repositories, create a separate project for each repo.
+- Is your app made up of several micro-services? Split those into projects accordingly.
+- If you have a monolithic codebase, set up separate projects for the Backend and Frontend.
+- Give each language its own project. If your backend code has NodeJS and Java components, separate those into two different projects.  
+
+## 5. Define Environments
+
+Depending on your development life cycle, your applications are probably deployed and running in multiple environments -  _Dev_, _Testing_, _Production_, etc. Configuring the environment in your Sentry SDK, will add the tag to every error event and will help you associate events to the environment in which they occurred. You can do that by creating a `SENTRY_ENVIRONMENT` environment variable or by explicitly defining it in the SDK configuration. For more information see our documentation on [Environments](https://docs.sentry.io/enriching-error-data/environments/?platform=browser).
```

Comment:
Technically dev/localhost errors shouldn't be going to sentry. let's remove mentions of dev